introduces the original paper, explains the technical details of your replication of analyses and summarizes your replication of the original results
Introduction to paper: "Cytochrome c oxidase subunit 1 barcode data of fish of the Nayband National Park in the Persian Gulf and analysis using meta-data flag several cryptic species" 

**Paper citation**: ASGHARIAN, H., SAHAFI, H.H., ARDALAN, A.A., SHEKARRIZ, S. and ELAHI, E. (2011), Cytochrome c oxidase subunit 1 barcode data of fish of the Nayband National Park in the Persian Gulf and analysis using meta-data flag several cryptic species. Molecular Ecology Resources, 11: 461-472. https://doi.org/10.1111/j.1755-0998.2011.02989.x

**Abstract**: We provide cytochrome c oxidase subunit 1 (COI) barcode sequences of fishes of the Nayband National Park, Persian Gulf,Iran. Industrial activities, ecological considerations and goals of The Fish Barcode of Life campaign make it crucial that fishspecies residing in the park be identified. To the best of our knowledge, this is the first report of barcoding data on fishesof the Persian Gulf. We examined 187 individuals representing 76 species, 56 genera and 32 families. The data flaggedpotentially cryptic species ofGerres filamentosusandPlectorhinchus schotaf. 16S rDNA data on these species are provided.Exclusion of these two potential cryptic species resulted in amean COI intraspecific distance of 0.18%, and a mean inter- tointraspecific divergence ratio of 66.7. There was no overlap between maximum Kimura 2-parameter distances among con-specifics (1.66%) and minimum distance among congenericspecies (6.19%). Barcodes shared among species were notobserved. Neighbour-joining analysis showed that most species formedcohesive sequence units with little variation.Finally, the comparison of 16 selected species from this studywith meta-data of conspecificsfrom Australia, India, Chinaand South Africa revealed high interregion divergences and potential existence of six cryptic species. Pairwise interregionalcomparisonsweremoreinformativethanglobaldivergenceassessments with regard to detection of cryptic variation. Ouranalysis exemplifies optimal use of the expanding barcode data now becoming available.

**Technical details of replication of analyses**

**Figure 3**
Daniel Moraes procedure
_Steps to get the sequences for Figure 3:_
_1- Obtain the sequences for Figure 3: There were 14 sequences in Figure 3_
In the Supporting information from the manuscript, there is a file called
“MEN_2989_sm_tS2.xls”
• In this file was possible to obtain only 6 sequences at the column named “Genbank
Num” by looking at the “Sample ID” described in the phylogenetic tree in Figure 3.
• These 6 sequences were from Argyrops spinifer specimens from Iran.
• Then we searched in NIH in search each gen bank number
https://www.ncbi.nlm.nih.gov/. For example, for Sample ID “NPPF1048” we entered
“HQ149794”.
• For the 8 remaining sequences, we entered each Sample ID in gen Bank. For example,
for Sample ID BW-675, we entered “Argyrops spinifer BW-A675”
• Then we saved the 14 sentences from FASTA format in a text file called “BCB 546 final
project-sequences-Figure 3.
_2- Import the sequences to MEGA- Molecular Evolutionary Genetic Analysis_
• The version from the manuscript was MEGA 4.0
• The current version that we used was MEGA 11
•
_3- The alignment was conducted by MUSCLE
4- To build the phylogenetic tree: Neighbour-joining (NJ) tree_

**Summarization of replication of orignial results**
